### PR TITLE
models: task, user, org models and association updated

### LINKED ISSUE #7: create sequelize task model

### LINKED ISSUE #8: create sequelize org model

### LINKED ISSUE #9: create sequelize user model

### PR DESCRIPTION
**Pull request by johnnyvcoding**
user, tasks, and org have associations created along with magic methods

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -1,6 +1,64 @@
...
 // register models
 require('./models')
 
+const UserOrganization = db.define('user_organization', {
+  role: {
+    type: Sequelize.ENUM,
+    values: ['owner', 'co-owner', 'user', 'admin']
+  },
+
+  OrgId: {
```
Comment (shapirodaniel) [inline server/db/index.js:16]: OrgId is a foreign key, the through table shouldn't define it explicitly

File: server/db/index.js [hunk 2]
```diff
@@ -1,6 +1,64 @@
...
+    type: Sequelize.INTEGER,
+    references: {
+      model: Organization,
+      key: 'id'
+    }
+
+  },
+
+
+  UserId: {
```
Comment (shapirodaniel) [inline server/db/index.js:26]: UserId is a foreign key, the through table shouldn't define it explicitly

File: server/db/index.js [hunk 3]
```diff
@@ -1,6 +1,64 @@
...
+  }
+
+})
+
+User.belongsToMany(Organization, {through: UserOrganization})
+Organization.belongsToMany(User, {through: UserOrganization})
+
+
+const UserTask = db.define('user_task', {
+  TaskId: {
```
Comment (shapirodaniel) [inline server/db/index.js:41]: if UserTask model doesn't have any fields unique to it then it's a composite key and we won't need to define task or user ids, they will be autogenerated by the association

File: server/db/models/organization.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const Organization = db.define('organization', {
+
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+
+  },
+
+  imageUrl: {
+    type: Sequelize.STRING,
```
Comment (shapirodaniel) [inline server/db/models/organization.js:15]: defaultValue rather than default

File: server/db/models/user.js [hunk 5]
```diff
@@ -3,11 +3,37 @@ const Sequelize = require('sequelize')
...
+
+
   email: {
     type: Sequelize.STRING,
     unique: true,
     allowNull: false
   },
+
+
+  imageUrl: {
```
Comment (shapirodaniel) [inline server/db/models/user.js:32]: defaultValue rather than default

**Review by shapirodaniel — CHANGES_REQUESTED** [review]
a few changes to through tables and defaultValue specs, otherwise looks great

**Review by shapirodaniel — APPROVED** [review]

**Review by TwelveEyes — APPROVED** [review]

**Pull request merged**